Dockerfile

```diff
@@ -13,31 +13,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.21.3@sha256:4d5cf6cb9269c5867ccf86b8282897b7eb4764d9739b1a19d8cc68643bbc3f3c AS builder
+FROM registry.access.redhat.com/ubi9/go-toolset@sha256:c3a9c5c7fb226f6efcec2424dd30c38f652156040b490c9eca5ac5b61d8dc3ca AS builder
 ENV APP_ROOT=/opt/app-root
 ENV GOPATH=$APP_ROOT
 
 WORKDIR $APP_ROOT/src/
 ADD go.mod go.sum $APP_ROOT/src/
-RUN go mod download
-
 # Add source code
 ADD ./ $APP_ROOT/src/
 
-RUN go build -o server main.go
-RUN CGO_ENABLED=1 go build -gcflags "all=-N -l" -o server_debug main.go
```

Comment:
Why are we removing the dubgging binary in the build process?

Reply:
Do we need it? There is no Red Hat documentation for it, and it wasn't used in any way in TP1. I think if we don't need to include it, let's don't just to keep the surface area as small as possible.